docs/installation-windows.md

```diff
@@ -0,0 +1,41 @@
+# Install epoch on Windows using the Windows Subsystem Linux
+
+NOTE: These steps describe the setup for a basic CPU mining configuration.
+      Windows Subsystem Linux has no GPU-passthrough support yet. Therefore,
+      a native Windows setup will be required to leverage GPU mining.
+
+## Prerequisites
+
+You need the following software:
+
+- An up-to-date Windows 10 installation which supports Windows Subsystem linux
+- A working Microsoft Store
+- PowerShell (which usually comes with Windows 10)
+
+## Step 1: Get helper scripts
+
+The helper scripts are located in the `scripts/` folder in the `epoch` Git repository. So you can get the scripts by
+
+1. Downloading the [Git repository](https://github.com/aeternity/epoch) itself;
```

Comment:
not the full git repository is needed, just 
`scripts/setup-windows-subsystem-linux.ps1`,
`scripts/run-miner-on-windows-subsystem-linux.ps1`
and `./setup-release-on-ubuntu.sh` with some optimization

Reply:
That is correct. However, I'd expect its more convenient for users to download one archive rather than 3 individual files. I'll make that clear in the section.